boa/src/syntax/lexer/token.rs

```diff
@@ -60,7 +60,7 @@ impl Display for Token {
 /// Represents the type differenct types of numeric literals.
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[derive(Clone, PartialEq, Debug)]
-pub enum NumericLiteral {
+pub enum Numeric {
```

Comment:
Why was this name changed? It's still a numeric literal, right?

Reply:
I think it is easier to understand named like this as it is wrapped inside a NumericLiteral as it seemed a bit weird having a numeric literal in a numeric literal

Reply:
Are you happy with this change or do you think it should be kept as NumericLiteral?